torch/fx/passes/_tensorify_python_scalars.py

```diff
@@ -261,10 +248,9 @@ def _sympy_interp(expr: sympy.Expr) -> MetaProxy:
 
                     node.replace_all_uses_with(replacement_proxy.node)
                     graph.erase_node(node)
-                    deleted.add(node)
 
     # DCE symbols (which are guaranteed to be pure) only
-    dce(graph, deleted)
+    graph.eliminate_dead_code()
```

Comment:
@bdhirsh do you know if it's safe to call `eliminate_dead_code()` on arbitrary FX graphs now. I still feel uncomfortable about the default predicate

Reply:
(Graph cannot be assumed to be functionalized)

Reply:
Why can't graphs can't be assumed to functionalized here? This pass is after functionalization and in our pass we only rewrite from TensorScalar to ScalarScalar and node.target(args) to node.target(args_with_symfloats_specialized)? Both of which won't introduce mutations?

Reply:
Oh you might be right lol.